internal/utils/url.go

```diff
@@ -89,3 +91,13 @@ func GetURLWithScheme(rawURL string) (string, error) {
 
 	return rawURL, nil
 }
+
+func GenerateTenantURL(baseURL string, tenantID string) (string, error) {
+	protocol, urlName, validURL := strings.Cut(baseURL, urlSeparator)
+	if !validURL || protocol == "" || urlName == "" {
+		return "", fmt.Errorf("invalid base URL: %s", baseURL)
```

Comment:
@marcelosalloum brought my attention before to the fact that `:` is used in golang for error wrapping. So it's best to avoid it and only use it in that case. I.e. `"this value is invalid: %w", err` 

https://github.com/stellar/stellar-disbursement-platform-backend/pull/230/files#r1562553009

Reply:
good point, I'll remove the colon